src/relay/QueryRenderer.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import * as React from 'react';
+import { QueryRenderer as OriginalQueryRenderer } from 'react-relay';
+
+import { UserContext, type UserContextType } from '../context/User';
+import { LanguageContext } from '../context/Language';
+import createEnvironment from './environment';
+import frontendLanguageToLocale from '../helpers/frontendLanguageToLocale';
+
+type RenderArgs<RenderProps> = {
+  error: ?Error,
+  props: ?RenderProps,
+};
+
+type Props<RenderProps> = {|
+  cacheConfig?: {| force: boolean |},
+  query: () => mixed,
+  render: (RenderArgs<RenderProps>) => ?React$Element<*>,
+  variables: Object,
+|};
+
+const QueryRenderer = <RenderProps>(props: Props<RenderProps>) => (
+  <UserContext.Consumer>
+    {({ loginToken }: UserContextType) => (
+      <LanguageContext.Consumer>
+        {language => {
+          const locale = frontendLanguageToLocale[language];
+
+          return (
+            <OriginalQueryRenderer
+              {...props}
+              environment={createEnvironment(loginToken, locale)}
```

Comment:
The imported function `createEnvironment` does not accept these two parameters as defined [here](https://github.com/kiwicom/smart-faq/blob/94d2fe5a7e9e517ace5049cb5c698f8dd8421348/src/relay/environment.js#L48). It should accept both token and locale

Reply:
🤦‍♂️

Reply:
fixed